.github/workflows/params-env.yaml

```diff
@@ -27,4 +27,3 @@ jobs:
       - name: Validate the 'manifests/base/params.env' file content
         run: |
           bash ./ci/check-params-env.sh
-
```

Comment:
thanks for lgtm @jstourac, last update. Format check was failing because of an extra blank line🙄 There is one failing check that fails as it doesn't allow my user to run the check, which should be overriden I assume